synapse/lib/lmdblayer.py

```diff
@@ -94,6 +100,29 @@ class LmdbLayer(s_layer.Layer):
             'range': self._rowsByRange,
         }
 
+    async def setNodeData(self, buid, name, item):
+        # TODO more str->utf8 caching
+        utf8 = name.encode('utf8')
```

Comment:
`'utf8'` is the default argument to str.encode() / str.decode(). We can remove it here.